nixos/modules/services/security/nginx-sso.nix

```diff
@@ -0,0 +1,56 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.nginx.sso;
+  pkg = getBin pkgs.nginx-sso;
+  configYml = pkgs.writeText "nginx-sso.yml" (builtins.toJSON cfg.configuration);
+in {
+  options.services.nginx.sso = {
+    enable = mkEnableOption "nginx-sso service";
+
+    configuration = mkOption {
+      type = types.attrsOf types.unspecified;
+      default = {};
+      example = {
```

Comment:
You need to do

```nix
{
  example = literalExample ''
    {
      listen = { addr = "127.0.0.1"; port = 8000; };
      # ...
    }
  '';
}
```

in order for it to show up correctly formatted in the docs

Reply:
Done.